mod/da/pkg/store/store.go

```
@@ -94,21 +97,67 @@
+		go func(index int, sc *types.BlobSidecar) {
+			defer wg.Done()
+
 			bz, err := sc.MarshalSSZ()
 			if err != nil {
-				return err
+				errChan <- err
+				return
+			}
+
+			// Store the sidecar
+			if err := s.IndexDB.Set(slot.Unwrap(), sc.KzgCommitment[:], bz); err != nil {
```

golangci-lint

shadow: declaration of "err" shadows declaration at line 120

(govet)